odxtools/nameditemlist.py

```diff
@@ -172,7 +172,7 @@ def __str__(self) -> str:
         return f"[{', '.join( [self._get_item_key(x) for x in self])}]"
 
     def __repr__(self) -> str:
-        return self.__str__()
+        return f"[{', '.join([repr(x) for x in self])}]"
```

Comment:
Make sure to include class name
f"NamedItemList[{', '.join([repr(x) for x in self])}]"

Reply:
fixed: 36d0ffff36ce29f28f999899de1c93572f877e3e